include/event_manager.h

```diff
@@ -21,105 +21,26 @@
 
 #include <zephyr.h>
 #include <zephyr/types.h>
-#include <sys/reboot.h>
 #include <sys/__assert.h>
-#include <logging/log_ctrl.h>
+#include <logging/log.h>
 
 #include <event_manager_priv.h>
-#include <profiler.h>
+
+#include <sys/reboot.h>
```

Comment:
Move it to `event_manager.c`. I think it should not be part of this header, because it's not related.